packages/aws-cdk-lib/aws-rds/lib/parameter-group.ts

```diff
@@ -83,6 +83,14 @@ export interface ParameterGroupProps {
    * @default - None
    */
   readonly parameters?: { [key: string]: string };
+
+  /**
+   * The CloudFormation policy to apply when the instance is removed from the
+   * stack or replaced during an update.
```

Comment:
Is this policy always needed or can we update the description to be more specific that it's only needed when the stack also has a `DatabaseInstance`?

Reply:
This doesn't have to be used in couple of DBInstance. It's technically a standalone removal policy that applies to ParameterGroup.